base/uk.ac.stfc.isis.ibex.e4.product/src/uk/ac/stfc/isis/ibex/e4/product/ApplicationWorkbenchWindowAdvisor.java

```diff
@@ -46,6 +56,19 @@
 
     private static final int MIN_WINDOW_HEIGHT = 800;
     private static final int MIN_WINDOW_WIDTH = 1100;
+    
+	private static final String DIALOG_BOX_TITLE = "IBEX is already running.";
+	private static final String DIALOG_QUESTION = "It appears that IBEX client is already running. "
+			+ "Are you sure you want to open another instance?";
+	
+	private static final String TEMP_PATH = System.getProperty("user.home") + "\\AppData\\Local\\IBEX\\tmp";
```

Comment:
If we want this path to be used as a generic temp path folder we should put it in /uk.ac.stfc.isis.ibex.preferences/src/uk/ac/stfc/isis/ibex/preferences/PreferenceSupplier.java and get it from there. Also we should do this in a cross-platform manner, see https://stackoverflow.com/questions/34459486/joining-paths-in-java for joining paths rather than using `\\`.